sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/TypeCoercionSuite.scala

```diff
@@ -1378,8 +1378,8 @@ class TypeCoercionSuite extends AnalysisTest {
     )
     ruleTest(inConversion,
       In(Literal("a"), Seq(Literal(1), Literal("b"))),
-      In(Cast(Literal("a"), StringType),
-        Seq(Cast(Literal(1), StringType), Cast(Literal("b"), StringType)))
+      In(Cast(Literal("a"), IntegerType),
```

Comment:
mmmh...honestly in this case I'd rather say that string is a better type for the cast than int. I am not sure which is the result of casting "a" and "b" to int...